### ISSUE TITLE
Time position indicator when pointer over Timeline is missing

### ISSUE THREAD

**Issue by SteveDaulton** [audacity/audacity#2122]
**Describe the bug**
When the pointer hovers over the Timeline, there should be a white vertical guide line across all tracks to clearly indicate the Quick Play position. The line does not appear

**To Reproduce**
1. Create a project with two or more tracks
2. Position the mouse pointer over the Timeline
3. Observe that there is no vertical white line over the tracks

**Expected behavior**
See screenshot

**Screenshots**

![indicator](https://user-images.githubusercontent.com/11681097/141372942-e4f337ba-9cbb-4075-9712-25725e3977cb.png)

**Additional information (please complete the following information):**
 - OS: Xubuntu 20.04
 - Version: Audacity 3.1.1

**Comment by petersampsonaudacity** [comment]
Testing this on W10 and mac OS 12.0.1 Monterey with 3.1.1:

1) on W10 is do still see the white line

2) on Mac I do **_NOT_** see the white line

**Comment by petersampsonaudacity** [comment]
On a related not: I observe that 3.1.x has lost the Timeline context menu command for disabling/enabling (on by default) command.

Is this a deliberate design decision, or just an error of ommission ?

**Comment by SteveDaulton** [comment]
Another user going back to an old version of Audacity because of this bug: https://forum.audacityteam.org/viewtopic.php?p=440414#p440414

**Comment by petersampsonaudacity** [comment]
Testing on W10 - this is a regression on 3.0.5 which was the last release that had the white line - marking this as P2 die to the regression (an based on the large number of complaints on the Forum - _many of which result in the user(s) reverting from 3.1 to an earlier Audacity version._

**Comment by petersampsonaudacity** [comment]
This is almost certainly brought about as a result of playback changes made for the new looping in 3.1.x

**Comment by lauriecooper** [comment]
I hope that this bug is soon fixed because I find this tool extremely useful. I'm syaying with 3.0.5 until it is fixed.

**Comment by crsib** [comment]
@Tantacrul

**Comment by Tantacrul** [comment]
Yes, the little white indicator line was an unfortunately overlooked casualty. We should definitely put it back in when the user is hovering over the timeline. In combination with the threshold work (see link), I think we should release another patch.
https://github.com/audacity/audacity/issues/2182#issuecomment-982517693

**Comment by petersampsonaudacity** [comment]
Testing on W10 with Paul's branch build  Audacity_Windows_64bit_1598995362_84704ab 

this looks to be fixed, tested with all four themes
![image](https://user-images.githubusercontent.com/50205984/146751085-6f09da47-f829-4bd7-9260-81b2dbb0fd86.png)

**Comment by lauriecooper** [comment]
Great news, thanks.

**Comment by petersampsonaudacity** [comment]
Tested on W10 with latest 3.1.3 alpha build Audacity_Windows_64bit_1602134390_7887ba0 

This looks good to go
